Patches/Core/ThingDefs_Races/Races_Animal_Giant.xml

```diff
@@ -192,13 +178,6 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="Thrumbo"]/race/baseHealthScale</xpath>
-		<value>
-			<baseHealthScale>5</baseHealthScale>
```

Comment:
The vanilla Thrumbo basehealthScale is only 0.7. Given the threat they're intended to present, I think the health and combat power patches for it should stay.